torch/autograd/gradcheck.py

```diff
@@ -458,6 +459,11 @@ def _differentiable_outputs(x):
     return tuple(o for o in _as_tuple(x) if o.requires_grad)
 
 
+def get_notallclose_msg(a, n, i, j, error_str='') -> str:
```

Comment:
nit: we can take this opportunity to improve these variable names, e.g.
a -> analytical
n -> numerical
i -> output_idx
j -> input_idx